data/profile.d/Makefile.am

```diff
@@ -16,6 +16,20 @@
 CLEANFILES = *~
 
 configdir           = $(sysconfdir)/$(PACKAGE_NAME)/profile.d
-dist_config_DATA    = *.conf
```

Comment:
Honestly, I'm not happy about this. I understand the reasoning in the article but for most of the situations they are not that problematic for Anaconda use-case. One of them is serious which is to avoid adding file which shouldn't be there, thought.

However, in general, the whole automake thing is just making our life worse and this is exactly the reason why. It's so easy in the Anaconda development to forget to add these because we are not testing the RPM for development but only parts in the code and we also do not have compilation which would show us the missing file.

Because of the above I would rather choose the edge here and use the wildcard here. We would like to get rid of the whole automake in the future so or so. @VladimirSlavik, @poncovka, @rvykydal, @M4rtinK  what do you think about that?

We even created a special test for not forgetting add files somewhere.

Reply:
It looks like our RPM tests should be able to detect unlisted `.conf` files, so both solutions are ok for me.

Reply:
Yes, I agree that in this case it makes more sense to grab all the configs. I also wonder about the discrepancy - @praiskup shows errors, while our builds succeed...

Reply:
I'm also fine with both solutions & totally agree with getting rid of autotools/automake for the main Anaconda package. 

AFAIK it's mostly used to build the Anaconda widgets, so maybe we could separate that into a separate package & project that keeps autotools/automake and build Anaconda by something more sane ? 

Even if we still needed to compile parts of isys I'm sure normal Python build tooling can handle a single native code extension just fine.